docat/docat/models.py

```diff
@@ -26,7 +26,13 @@ class ProjectVersion(BaseModel):
 class Project(BaseModel):
     name: str
     logo: bool
-    versions: list[ProjectVersion]
+
+    """
+    This is an approximations, because we don't actually check
+    if the versions are hidden and so on. It's just a placeholder,
+    the exact number is then loaded afterwards, to reduce the initial page load.
+    """
+    version_count: int
```

Comment:
I don't quite get why this change was done, why don't we just return the versions?

Reply:
I did it because I thought that this could  be our performance bottleneck. But I can try out how well it works with just the tooltips changed.